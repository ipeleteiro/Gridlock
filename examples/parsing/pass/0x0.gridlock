GRIDLOCK
Players: Amy, Bill.
Grid: 0x0.
Colours: red, green, blue.
Amy cannot move.
Bill wins!