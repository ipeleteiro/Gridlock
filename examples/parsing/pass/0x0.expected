GameRecord {players = ["Amy","Bill"], grids = [Grid {width = 0, height = 0, rep = fromList []}], colours = fromList [Red,Green,Blue]}